src/code.sh

```
@@ -71,82 +83,73 @@ collect_rnaseq_metrics() {
+        collect_hs_metrics
+    fi
+
+    if [[ "$run_CollectTargetedPcrMetrics" == true ]]; then
+        collect_targeted_pcr_metrics
+    fi
+
+    if [[ "$run_CollectRnaSeqMetrics" == true ]]; then
+        # Create refFlat file if not provided by user
+        if [ -z "$ref_annot_refflat" ]; then
+            echo "No refFlat file provided - creating GTF from refFlat file in CTAT bundle"
+            lib_dir=$(echo $fasta_index_name | cut -d "." -f 1,2)
```

Shellcheck (0.10.0)

[warning] fasta_index_name is referenced but not assigned.

(SC2154)